doc/nrf/ug_bootloader.rst

```diff
@@ -7,212 +7,198 @@ Secure bootloader chain
+The immutable bootloader supports two partition slots in which the upgradable bootloader can be stored: slot 0 and slot 1.
+Immutable bootloaders always update the upgradable bootloader using the :doc:`direct-xip <mcuboot:design>` strategy: placing a new bootloader image in the slot that is not in use by the current version of the upgradable bootloader.
 When booting, the immutable bootloader checks the version information for the images in slot 0 and slot 1 and boots the one with the highest version.
-If this image is faulty and cannot be booted, the other partition will always hold a working image and be booted instead.
-
-Set the option :option:`CONFIG_BUILD_S1_VARIANT` when building the upgradable bootloader to automatically generate pre-signed variants of the image for both slots.
-These signed variants can be used to perform an upgrade procedure through the :ref:`lib_fota_download` library.
-
-.. _ug_bootloader_adding:
-
-Adding a bootloader chain to your application
-*********************************************
-
-The |NCS| includes a sample implementation of an :ref:`immutable bootloader <bootloader>`.
-Additionally, the |NCS| comes with a slightly modified version of :doc:`mcuboot:index`.
-
-Both bootloaders can easily be included in your application using :ref:`multi-image builds <ug_multi_image>`.
-
-Adding the immutable bootloader
-===============================
-
-To add the immutable bootloader to your application, set :option:`CONFIG_SECURE_BOOT` and add your private key file under :option:`CONFIG_SB_SIGNING_KEY_FILE`.
-|how_to_configure|
-
-The :ref:`bootloader_build_and_run` section has detailed instructions for adding the immutable bootloader as first stage of the secure bootloader chain.
-
-To ensure that the immutable bootloader occupies as little flash as possible, apply the :file:`overlay-minimal-size.conf` Kconfig overlay file for the b0 image.
-This can be done in the following way:
-
-* Using cmake::
-
-     cmake -GNinja -DBOARD=nrf52840dk_nrf5840 -Db0_OVERLAY_CONFIG=overlay-minimal-size.conf -DCONFIG_SECURE_BOOT=y ../
+If this image is faulty and cannot be booted, the other partition will always hold a working image that will be booted instead.
 
-* Using west::
+The :ref:`bootloader capabilities table <app_bootloaders_support_table>` details which bootloaders are valid to use as an upgradable bootloader.
 
-     west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- -Db0_OVERLAY_CONFIG=overlay-minimal-size.conf -DCONFIG_SECURE_BOOT=y
+.. _upgradable_bootloader_presigned_variants:
 
+Pre-signed variants
+-------------------
 
-Adding MCUboot as an upgradable bootloader
-==========================================
+When programming an upgradable bootloader, it is possible to automatically generate pre-signed variants of the image for both slots.
+This is to ensure that the update candidate is linked correctly against both image slots, as it is not guaranteed to be executed from one particular slot.
```

Comment:
```suggestion
An OTA update package of the upgradable bootloader contains both images, since it might be unknown which slot is available for upgrade.
```